### PR TITLE
add location setting for national cloud in sample setting files and enable CI for mooncake

### PR DESCRIPTION
**Pull request by VincentLiu028**
Add location setting in sample configuration files to support national clouds. The default value is empty, so public cloud is used. For national cloud, use the data center name of national cloud. For China the value is chinaeast or chinanorth

---
This change is [Reviewable](https://reviewable.io/reviews/azure/acs-engine/732)

### CODE REVIEW

File: test/cluster-tests/dcos/test.sh [hunk 1]
```diff
@@ -15,16 +15,20 @@ DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
 set -x
 
 source "$DIR/../utils.sh"
+<<<<<<< HEAD
```
Comment (rjtsdl) [inline test/cluster-tests/dcos/test.sh]: What's this?

**Comment by rjtsdl** [comment]
Btw, "location" is one of the reserved field by ARM, it is the Azure region. It cannot be customized. 
If your change is compatible with ARM's specs, it is fine. Otherwise, please come up with something else.

File: test/cluster-tests/dcos/test.sh (hunk 1 above)
Comment (VincentLiu028) [inline test/cluster-tests/dcos/test.sh]: forget to remove them. merge the changes.

**Comment by VincentLiu028** [comment]
location setting is added to profile file, not arm file. The location setting will be translate to deployment target. Then based on different deployment target, the different endpoints will be used. All these logic already existed in acs-engine. Only add location setting to open this configuration. So we could deploy acs-engine on national cloud and trigger CI.

**Comment by shrutir25** [comment]
@rjtsdl - how do we proceed with this PR ?

**Comment by seanknox** [comment]
@shrutir25 @VincentLiu028  Last thing to fix is the git cruft `test/cluster-tests/dcos/test.sh`

File: test/cluster-tests/dcos/test.sh (hunk 1 above)
Comment (seanknox) [inline test/cluster-tests/dcos/test.sh]: Need you to remove this before we can proceed.

Comment (VincentLiu028) [inline test/cluster-tests/dcos/test.sh]: Fix it.

**Comment by shrutir25** [comment]
@VincentLiu028 - we will be closing this PR since there has been no response for over 2 weeks. Please feel free to reopen it.

**Pull request closed without merging**